### ISSUE TITLE
Benchmark instruction not working

### ISSUE THREAD

**Issue by elimist** [nats-io/nats-site#82]
I'm trying to follow the benchmark instructions [here](http://nats.io/documentation/tutorials/nats-benchmarking/) and the java steps seem to fail with lots of "cannot find symbol" errors. 

I don't know anything about Java, but I was able to compile the source files in step 1 by specifying a sourcepath and classpath:

`javac -d ./bin -sourcepath ./src/main/java/org/nats/*.java -cp /Users/x/.m2/repository/org/slf4j/slf4j-api/1.7.14/slf4j-api-1.7.14.jar`

However now on the next step `javac -d ./bin ./src/test/java/org/nats/benchmark/*.java`

I'm getting additional errors:

```
./src/test/java/org/nats/benchmark/PubPerf.java:17: error: cannot find symbol
                Connection conn = Connection.connect(prop);
                ^
  symbol:   class Connection
  location: class PubPerf
./src/test/java/org/nats/benchmark/PubPerf.java:17: error: cannot find symbol
                Connection conn = Connection.connect(prop);
                                  ^
  symbol:   variable Connection
  location: class PubPerf
./src/test/java/org/nats/benchmark/PubSubPerf.java:18: error: cannot find symbol
                final Connection conn1 = Connection.connect(new Properties());
                      ^
  symbol:   class Connection
  location: class PubSubPerf
./src/test/java/org/nats/benchmark/PubSubPerf.java:18: error: cannot find symbol
                final Connection conn1 = Connection.connect(new Properties());
                                         ^
  symbol:   variable Connection
  location: class PubSubPerf
./src/test/java/org/nats/benchmark/PubSubPerf.java:19: error: cannot find symbol
                final Connection conn2 = Connection.connect(new Properties());
                      ^
  symbol:   class Connection
  location: class PubSubPerf
./src/test/java/org/nats/benchmark/PubSubPerf.java:19: error: cannot find symbol
                final Connection conn2 = Connection.connect(new Properties());
                                         ^
  symbol:   variable Connection
  location: class PubSubPerf

```

I also tried specifying various combinations of classpath/sourcepath, but it didn't work. I'm guessing javac just needs to know the classpath for Connection, but i'm not too sure where that is. Is there an easy way of getting the benchmarks up and running?

**Comment by mcqueary** [comment]
Hi. I'm very sorry for the inconvenience,. The instructions you are referencing are outdated and reference a third party library that is not supported by Apcera. I'll take a look at updating this with the more current information on Monday.

**Comment by elimist** [comment]
Ah, thanks for the clarification. Looking forward to it!

**Comment by mcqueary** [comment]
Closed by #173 -- once the changes are published to the site, you'll see instructions for using the Go-based `nats-bench` that is included in the `nats-io/nats` package. 

http://nats.io/documentation/tutorials/nats-benchmarking/
